### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by l3r8yJ**
here some improvements, @h1alexbel take a look, please 
I also moved the fake package to the test directory because I think it was created for testing.

### CODE REVIEW

File: src/test/java/fake/FkProducer.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * SOFTWARE.
  */
 
-package org.eocqrs.kafka.fake;
+package fake;
```
Comment (h1alexbel) [inline src/test/java/fake/FkProducer.java:23]: The idea was to build fake producers, consumers to use them as [Fake objects](https://www.yegor256.com/2014/09/23/built-in-fake-objects.html) in tests, in order to don't mock. Not only in our project, but in projects where we will add eo-kafka dependency.

Like jcabi-github does: https://github.com/jcabi/jcabi-github/tree/master/src/main/java/com/jcabi/github/mock

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

**Review by h1alexbel — APPROVED** [review]

**Comment by 0pdd** [comment]
@l3r8yJ the puzzle [#47](https://github.com/eo-cqrs/eo-kafka/issues/47) is still not solved.

**Comment by h1alexbel** [comment]
@l3r8yJ thanks!

**Comment by 0pdd** [comment]
@l3r8yJ the puzzle [#98](https://github.com/eo-cqrs/eo-kafka/issues/98) is still not solved; solved: [#47](https://github.com/eo-cqrs/eo-kafka/issues/47).

**Comment by 0pdd** [comment]
@l3r8yJ the puzzle [#111](https://github.com/eo-cqrs/eo-kafka/issues/111) is still not solved; solved: [#47](https://github.com/eo-cqrs/eo-kafka/issues/47), [#98](https://github.com/eo-cqrs/eo-kafka/issues/98).

**Comment by 0pdd** [comment]
@l3r8yJ all 3 puzzles are solved here: [#111](https://github.com/eo-cqrs/eo-kafka/issues/111), [#47](https://github.com/eo-cqrs/eo-kafka/issues/47), [#98](https://github.com/eo-cqrs/eo-kafka/issues/98).

**Pull request merged**